tests/installation/ipxe_install.pm

```diff
@@ -170,12 +168,13 @@ sub run {
             record_info("TW install?", "Pls make sure it is Tumbleweed that is being installed.", result => 'softfail');
         }
         assert_screen([qw(network-config-created loading-installation-system)], 60);
+        return if get_var('AUTOYAST');
         wait_still_screen(stilltime => 12, similarity_level => 60, timeout => 30) unless check_screen('sshd-server-started', timeout => 60);
         save_screenshot;
     }
 
     # when we don't use autoyast, we need to also load the right test modules to perform the remote installation
-    if (get_var('AUTOYAST') and !get_var('VIRT_AUTOTEST')) {
+    if (get_var('AUTOYAST')) {
```

Comment:
does this still make sense here? this condition wont never run due to https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17153/files#diff-f8329f3474ec62c7e8b241c5be029319db687b91e770ec1f2327dc96ed9c0a97R171, right?

Reply:
No, `return if get_var('AUTOYAST');` is in the condition block `if (get_var('VIRT_AUTOTEST') && is_tumbleweed) {`.
So the PR does not change the logic of tests without `VIRT_AUTOTEST=1`